### ISSUE TITLE
catch up on books

### ISSUE THREAD

**Issue by chadwhitacre** [gratipay/finances#3]
This is a meta-ticket for all the months we need to catch up on, going back to the beginning.

---
- [x] 2012-06—#1
- [ ] 2012-07—#2
- [ ] 2012-08—#6
- [ ] 2012-09
- [ ] 2012-10
- [ ] 2012-11
- [ ] 2012-12
- [ ] 2013-01
- [ ] 2013-02
- [ ] 2013-03
- [ ] 2013-04
- [ ] 2013-05
- [ ] FY2013

---
- [ ] 2013-06
- [ ] 2013-07
- [ ] 2013-08
- [ ] 2013-09
- [ ] 2013-10
- [ ] 2013-11
- [ ] 2013-12
- [ ] 2014-01
- [ ] 2014-02
- [ ] 2014-03
- [ ] 2014-04
- [ ] 2014-05
- [ ] FY2014

---
- [ ] 2014-06
- [ ] 2014-07
- [ ] 2014-08
- [ ] 2014-09
- [ ] 2014-10
- [ ] 2014-11
- [ ] 2014-12
- [ ] 2015-01
- [ ] 2015-02
- [ ] 2015-03
- [ ] 2015-04
- [ ] 2015-05
- [ ] FY2015

---
- [ ] 2015-06
- [ ] 2015-07
- [ ] 2015-08
- [ ] 2015-09
- [ ] 2015-10
- [ ] 2015-11
- [ ] 2015-12
- [ ] 2016-01—#14
- [ ] 2016-02
- [ ] 2016-03
- [ ] 2016-04
- [ ] 2016-05
- [ ] FY2016

---
- [ ] 2016-06
- [ ] 2016-07
- [ ] 2016-08
- [ ] 2016-09
- [ ] 2016-10
- [ ] 2016-11—#30
- [ ] 2016-12—#31
- [ ] 2017-01—#44
- [ ] 2017-02
- [ ] 2017-03
- [ ] 2017-04
- [ ] 2017-05
- [ ] FY2017

**Comment by chadwhitacre** [comment]
@kaguillera and I are thinking that we should do a review/audit after each year, rather than waiting until we're totally caught up and reviewing four years at once.

**Comment by chadwhitacre** [comment]
to: CPA

> Thanks for putting us on the right track this morning, Peter. Good to be able to proceed with confidence. For the record, here are [our meeting notes](https://github.com/gratipay/inside.gratipay.com/issues/350#issuecomment-176242898). We may have a question or two as we implement our plan, I'll follow up as necessary.
> 
> One immediate question: Kwesi and I were thinking we should review or audit a year at a time rather than waiting to audit all four years of backlog at once. Does that sound right to you?

**Comment by chadwhitacre** [comment]
![tumblr_m6p5v8xlyd1rn81gb](https://cloud.githubusercontent.com/assets/134455/12658199/922eb728-c5d5-11e5-88be-304cf47551aa.jpg)

**Comment by chadwhitacre** [comment]
> Are we talking about a CPA review/audit per year?  If so, I would rather you save on the charges if not necessary.  Any third party will inquire only for most recent audit/review as opposed to historical.  Unless you must do this to satisfy a grant or other need, then let’s discuss having to do this.

---

> Thanks for the recommendation. We have two main goals here:
> 1. To assure ourselves that we haven't mismanaged our escrow.
> 2. To demonstrate our soundness to current and potential customers.
> 
> I'm open to the idea that we don't need an audit for past years in order to accomplish our goals, though we'd need an estimate in order to look at the cost/benefit.
> 
> Of course, at our current pace, we may have FY2016 or even FY2017 under our belts before we finish reconciling FY2013 (by the way, we decided to go with a June 1 fiscal year, to sync with the start of our books and set us up for an off-season audit). In that case, it'd be easy to decide to start with the later year.
> 
> Perhaps we can revisit the question when we get closer to having our first full year reconciled one way or another?

**Comment by chadwhitacre** [comment]
> Sounds good Chad.  Thanks

**Comment by chadwhitacre** [comment]
We _gotta_ get this done, sooner rather than later. We want to run an audit for FY2016 mid-year. We _have_ to be caught up by then. @kaguillera and I are now thinking that we should _not_ try to start on January 2016 until we've caught up with the rest first. Otherwise we're going to be bouncing back and forth. I think it's going to be more efficient to make a straight run through. Over at https://github.com/gratipay/inside.gratipay.com/issues/308#issuecomment-142399629 we collected most of the CSVs we need. Now we're working on getting them into a standard format in the `logs` repo, and then we can start writing scripts to convert raw CSVs from banks and processors into `dat` ledgers.

**Comment by chadwhitacre** [comment]
Paris (https://github.com/gratipay/inside.gratipay.com/issues/314) is just before the end of our fiscal year (May 31). Let's make that our target to get this done, so that we can schedule our first audit for June/July. That gives us 15 weeks to get `((13 * 4) - 1) =` 51 PRs landed: three or four PRs per week.

**Comment by chadwhitacre** [comment]
New Alliance CSVs updated in https://github.com/gratipay/logs/commit/d26ee72b2381490f3772890529b6c3cc248e0890.

**Comment by techtonik** [comment]
Last link is 404.

**Comment by chadwhitacre** [comment]
@techtonik That's because you don't have perms on the `logs` repo.

**Comment by nobodxbodon** [comment]
Seems there was a lot progress in [#308](https://github.com/gratipay/inside.gratipay.com/issues/308#issuecomment-142399629). Beside doing catchup, is there plan to start book-keeping for current transactions? Or is it already setup?

**Comment by chadwhitacre** [comment]
Yeah, we made a big push on this earlier, but then got side-tracked before really landing it. We really should get back to this. Perhaps you and I could meet up on Saturday or Sunday afternoon (November 12 or 13) to come up with a plan? There is a lot going on here ...

**Comment by nobodxbodon** [comment]
Could I start working on this month's booking (1/2017)? Even with https://github.com/gratipay/finances/pull/35 not finished yet, I can still start getting familiar into the new transactions. I'll still work on pre-2017 booking, but IMO the latest is the most critical.

I will need the cleaned-up (masked account #, etc) bank statements or other material.

**Comment by nobodxbodon** [comment]
@whit537 as mentioned in https://github.com/gratipay/finances/issues/3#issuecomment-275812805, shall I start with log or maybe it's simpler for you to export csv from bank statements?

**Comment by nobodxbodon** [comment]
@whit537 now that I think of it, is it even practical to skip all the previous months and work on 01/2017? Even with the starting balance of New Alliance?

**Comment by chadwhitacre** [comment]
We would have to enter starting balances for any accounts in use in 2017-01, yes. Which month would you like to tackle next? 2012-07 (#2) or 2017-01?

**Comment by nobodxbodon** [comment]
If feasible, I prefer to tackle 2017-01 first, hopefully keep book-keeping in future months, and going backwards when time allows.

**Comment by nobodxbodon** [comment]
@whit537 as I've setup freshdesk access through https://github.com/gratipay/inside.gratipay.com/issues/878#issuecomment-278486328 (thanks!), may I confirm if below are the steps to tackle 2017-01:

1. search for all the receipts within 2017-01 in freshdesk
2. put starting balances (you will provide?) in FY2017/FY2017.beancount, like https://github.com/gratipay/finances/blob/master/FY2013/FY2013.beancount#L7
3. create FY2017/2017-01.beancount based on (1) and statements in logs

Is there necessary to pull some numbers from database?

**Comment by chadwhitacre** [comment]
Yes, though for (2) you should be able to get opening balances from the logs repo, no?

> Is there necessary to pull some numbers from database?

Oh, probably. :-) I think we may need to bring back the detailed part of the reconciliation dashboard in order to do paydays.

**Comment by nobodxbodon** [comment]
> Yes, though for (2) you should be able to get opening balances from the logs repo, no?

Yes for PNC, no for New Alliance. BTW new-alliance.csv has a empty line after header, otherwise github can render the file in table format.

> Oh, probably. :-) I think we may need to bring back the detailed part of the reconciliation dashboard in order to do paydays.

If that's a blocker, is there an issue for this already? If not could you open one with some context please?

**Comment by kaguillera** [comment]
Think I should start working on https://github.com/gratipay/finances/pull/2 once https://github.com/gratipay/gratipay.com/pull/4128 is merged since is working on the present month.

**Comment by nobodxbodon** [comment]
To [track paypal activities](https://github.com/gratipay/finances/pull/38#issuecomment-278882426), I suppose opening balance for paypal 2017-01 is also needed?

**Comment by chadwhitacre** [comment]
I guess this and subordinate PRs are now blocking on https://github.com/gratipay/finances/issues/43. 😩
